docs/release-notes.mdx

```diff
@@ -8,6 +8,19 @@ import { TitleCard } from './components/TitleCard';
     Keep up to date with the latest releases of the design system.
 </TitleCard>
 
+## 1.25.2 (Unreleased)
```

Comment:
Since this adds new APIs I would consider this a minor version change to either change the title to `1.26.0` (Unreleased) or just `Unreleased`.